packages/back-end/types/audit.d.ts

```diff
@@ -46,6 +46,10 @@ export type EventType =
   | "organization.create"
   | "organization.update"
   | "organization.delete"
+  | "environment.create"
+  | "environment.update"
+  | "environments.update"
+  | "environment.delete"
```

Comment:
Do you also need an entry in `EntityType`?

Reply:
When doing the bulk updates - either via the `putEnvironmentOrder` or `putEnvironments` - there isn't a single env.id for the entity, so I've currently just got it set to the organization. Not sure if that's the right direction...

Reply:
I have a PR changing those type definitions to take a constant object mapping entities to audit event names. This should fix these kind of gaps.